CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
+[Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+
+## [0.4.3] - Unreleased
+
++ Add - pre-commit checks to the repo to observe flake8, black, isort
++ Add - str_to_bool utility
```

Comment:
I question whether this function is needed. It does not seem substantial enough for common use. Just implement this where you use it.